ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -383,17 +373,44 @@ Rectangle {
         }
     }
 
+    //mentions helper properties
+    property int copyTextStart: 0
+    property string copiedTextPlain: ""
+    property string copiedTextFormatted: ""
+    ListView {
+        id: dummyContactList
+        model: control.usersStore.usersModel
+        delegate: Item {
+            property string contactName: model.name
+        }
+    }
+
     function onRelease(event) {
         if (event.key === Qt.Key_Backspace && textFormatMenu.opened) {
             textFormatMenu.close()
         }
         // the text doesn't get registered to the textarea fast enough
         // we can only get it in the `released` event
+
         if (paste) {
+            if (copiedTextPlain.includes("@")) {
+                copiedTextFormatted = copiedTextFormatted.replace(/underline/g, "none").replace(/span style="/g, "span style=\" text-decoration:none;");
+                for (var j = 0; j < dummyContactList.count; j++) {
+                    var name = dummyContactList.itemAtIndex(j).contactName;
+                    if (copiedTextPlain.indexOf(name) > -1) {
+                        name += '';
+                        copiedTextPlain += '';
```

Comment:
These two lines don't seem to do much. Are they to convert the variables to string?

Reply:
Removed